masterlist.yaml

```
@@ -11476,6 +11476,9 @@ plugins:
     msg:
       - <<: *alsoUseSilentVoice
         condition: 'not file("OBSE/Plugins/Elys_USV.dll") and not file("OBSE/Plugins/voicefile_redirector.dll") and not file("Woodland Rangers - SilentVoices.bsa")'
+      - <<: *wildEdits
+        subs: [ 'Remove the **Worldspace Sub-Block 3, 3 in Block 0, 0** from **Woodland Rangers.esp**.' ]
```

The message can be improved a bit to denote which worldspace to remove it from:

```suggestion
        subs: [ 'Remove **Sub-Block 3, 3** in **Block 0, 0** of **0000003C** from **Woodland Rangers.esp**.' ]
```